src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (bet1 == result1) & (result2 == bet2)
+                ? 2 :
+                (result1 > result2) & (bet1 > bet2)
```

Comment:
```suggestion
                (result1 > result2) && (bet1 > bet2)
```
can we use && here, or is it still better to use & ?

Reply:
До мене нарешті дійшло) Виправила